src/panel-triggers/components/AlertList/AlertCard.tsx

```diff
@@ -72,7 +72,7 @@ export default class AlertCard extends PureComponent<AlertCardProps, AlertCardSt
     let problemColor: string;
     if (problem.value === '0') {
       problemColor = panelOptions.okEventColor;
-    } else if (panelOptions.markAckEvents && problem.acknowledges && problem.acknowledges.length) {
+    } else if (panelOptions.markAckEvents && problem.lastEvent.acknowledged == "1") {
```

Comment:
Should be strict `===`
```suggestion
    } else if (panelOptions.markAckEvents && problem.lastEvent.acknowledged === "1") {
```